ParisBNF/et/BNFet116.xml

```diff
@@ -193,61 +244,61 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+                     <handNote xml:id="h1" script="Ethiopic">
+                        <date notBefore="1700" notAfter="1799"/>
                      </handNote>
-                  </handDesc> 
-                  
+                  </handDesc>
+
                   <additions>
                      <list>
-               
-                        
+
+
                         <item xml:id="a1">
                            <locus from="1" to="5"/>
-                           <desc type="GuestText">Several hymns to <persName ref="PRS6819Mary"/> and <persName ref="PRS5684JesusCh"/> have been added later.</desc> 
+                           <desc type="GuestText">Several hymns to <persName ref="PRS6819Mary"/> and <persName ref="PRS5684JesusCh"/> have been added later.</desc>
                            <!--is this a guest text?-->
                         </item>
-                        
+
                         <item xml:id="a2">
                            <locus from="46v" to="47"/>
-                           <desc type="GuestText">Several hymns to <persName ref="PRS6819Mary"/> and <persName ref="PRS5684JesusCh"/> have been added later.</desc> 
+                           <desc type="GuestText">Several hymns to <persName ref="PRS6819Mary"/> and <persName ref="PRS5684JesusCh"/> have been added later.</desc>
                            <!--is this a guest text?-->
                         </item>
-                        
+
                         <item xml:id="a3">
                            <locus from="80" to="81"/>
-                           <desc type="GuestText">Several hymns to <persName ref="PRS6819Mary"/> and <persName ref="PRS5684JesusCh"/> have been added later.</desc> 
+                           <desc type="GuestText">Several hymns to <persName ref="PRS6819Mary"/> and <persName ref="PRS5684JesusCh"/> have been added later.</desc>
                            <!--is this a guest text?-->
                         </item>
-                        
+
                         <item xml:id="a4">
                            <locus target="#48"/>
-                           <desc type="MagicFormula">Magic formula against hail.</desc> 
+                           <desc type="MagicFormula">Magic formula against hail.</desc>
                         </item>
-                        
+
                         <item xml:id="e1">
                            <desc>The name of <persName ref="PRS11217AdfaD"/> is also mentioned at the end of <ref target="#ms_i3"/>. </desc>
                            <q xml:lang="gez"/>
                         </item>
-                        
+
                      </list>
                   </additions>
                </physDesc>
-            
+
                <history>
                   <origin>
-                     <origDate notBefore="1500" notAfter="1599" evidence="lettering"/>
+                     <origDate notBefore="1700" notAfter="1799" evidence="lettering"/>
```

Comment:
Maybe, in the case of such stark change from the catalogue estimation, it would be good to add a `@resp` to the date, or possible even a short sentence? I think users would not be able to tell whether it's the catalogue's or your updated date now.

Reply:
@DenisNosnitsin1970 I think @thea-m is waiting for this small update before she approves and we can close this one